Added info about ability to overwrite (#29944)
elastic/beats: libbeat/processors/actions/docs/add_fields.asciidoc

++++

The `add_fields` processor adds additional fields to the event.  Fields can be
scalar values, arrays, dictionaries, or any nested combination of these. 

> The add_fields processor does have the ability to overwrite taget field if it already exists. 

Reviewer: ```suggestion
The `add_fields` processor will overwrite the target field if it already exists. 
```
